### PR TITLE
License backend

### PR DESCRIPTION
**Pull request by iartem**

### CODE REVIEW

File: frontend/express/app.js [hunk 1]
```diff
@@ -916,6 +931,8 @@ Promise.all([plugins.dbConnection(countlyConfig), plugins.dbConnection("countly_
                 path: countlyConfig.path || "",
                 cdn: countlyConfig.cdn || "",
                 message: req.flash("message"),
+                notify,
+                license,
```
Comment (iartem) [inline frontend/express/app.js:935]: @kanwarujjaval license is an error message for frontend. It's empty if we're all good, `license.err.restricted` if instance is locked (only license manager for global admin case) and `license.err.locked` if instance is restricted (prevent login for non global admins case).

File: plugins/server-stats/api/jobs/stats.js [hunk 2]
```diff
@@ -141,6 +141,17 @@ class StatsJob extends job.Job {
                                 });
 
                                 Countly.userData.save();
+
+                                Countly.fetch_remote_config(null, null, (e, conf) => {
+                                    if (err) {
+                                        log.e('Failed to fetch remote config', e);
+                                    }
+                                    else {
+                                        db.collection('plugins').updateOne({_id: 'plugins'}, {$set: {remoteConfig: conf || {}}}).catch(dbe => {
```
Comment (iartem) [inline plugins/server-stats/api/jobs/stats.js:150]: @kanwarujjaval remote config is fetched during regular stats ping and stored in db, let me know if it won't work

**Review by iartem — COMMENTED** [review]
Check comments above please

**Review by iartem — COMMENTED** [review]
Check comments above please

**Review by kanwarujjaval — APPROVED** [review]

**Pull request merged**